about.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Asia Web Conference 2022</title>
+  <link rel="stylesheet" href="./style-about.css">
+  <link rel="shortcut icon" href="./assets/AsiaWebConf-Icon.png" type="image/x-icon">
+  <script src="https://kit.fontawesome.com/41c9961a42.js" crossorigin="anonymous"></script>
+  <script src="./script.js" defer></script>
+</head>
+
+<body>
+  <!-- Heading Section -->
+  <div class="bgContainer" id="home">
+    <div class="menuBar">
+      <button class="menu" type="button">
+        <img src="./assets/hamburger.svg" alt="">
+      </button>
+    </div>
+    <nav>
+      <div class="iconBar">
```

Comment:
- [ ] Please, consider that moving the `Home` link in the mobile menu to the top will be more correct, as the `Home` page comes first.
<img width="389" alt="Screen Shot 2022-04-12 at 10 31 44" src="https://user-images.githubusercontent.com/49064106/162895837-3e9506ab-eab0-446e-a86e-1e0e3197ea1c.png">

Reply:
@elmar8287 Yes, rearrange already.